src/ms.tex

```diff
@@ -603,45 +603,42 @@ \subsection{Project governance} \label{sec:project-governance}
 %Briefly summarize new procedures and governance structure, new CoCo, election
 %process overview, etc.
 
-As part of the process of developing \astropypkg into a long-term sustainable
+As part of the process of developing \astropy into a long-term sustainable
 product, and to improve transparency and accountability, the Project agreed to
 write down and formalize our governance structure (partly supported by explicit
 funding for this purpose; see Section~\ref{sec:project-funding}). At the 2019
 Astropy Coordination Meeting, input was gathered from participants on what governance
 structures existed in the associated Open Source Software communities, and what
-would fit well with the needs of \astropy. This led into a ``retreat'' planned
+would fit well with the needs of \astropy. This led to a ``retreat'' planned
 for March 2020, but due to the COVID-19 pandemic, this became a series of
 virtual meetings of the ``Astropy Governance Working Group.'' This group
-drafted the APE 0 document \citep{ape0}, which was then eventually ratified and
+drafted the APE 0 document \citep{ape0}, which was eventually ratified and
 implemented by the ``Astropy Governance Implementation Working Group'' in Fall
-2021. While the process emphasized flexibility and the ability to adapt to
-changing circumstances, it is expected that this is the framework Astropy's
-governance will operate in for at least the medium-term future.
+2021. While the process emphasized flexibility and adaptability,
+it is the expectation that \astropy's governance will operate in
+this framework for at least the medium-term future.
 
 The APE 0 \citep{ape0} document lays out the principles of this governance
 structure, so we refer the reader to that document for a more thorough
-description, and here we only highlight some key elements. While many of these
-principles were already de facto true or have been discussed organically (and
-have been discussed in earlier papers in this series), the APE 0-based governance
-aims to provide a single place where the community can agree as a starting
-point. With this in mind, it highlights the developer and user community of the
-Astropy Project as the ultimate sources of authority, as well as the core
-principle of ``do-ocracy'': those who do work for the Project (be it coding,
+description, and here we only highlight some key elements. The APE 0-based
```

Comment:
I try to shorten this section a little as #85 says this paper is too long but hopefully I didn't change its meaning.

Reply:
This works. 👍